meson.build

```diff
@@ -0,0 +1,2 @@
+project('boostut', 'cpp')
+boostut_dep = declare_dependency(include_directories : include_directories('include'))
```

Comment:
how can we test it? we should we add meson travis integration?

Reply:
Using meson with travis seems to be possible by doing the following.

- [Continuous Integration - The Meson Build system](https://mesonbuild.com/Continuous-Integration.html#travis-without-docker)

After that, we just need to configure a minimal meson C++ project and `meson build` or It seems.

Will the project structure look like this (In any directory):

- `test.cpp` (empty file)
- `subprojects` (directory)
    - `boost.ut.wrap` (file)
    ```
    [wrap-git]
    directory=ut
    url=https://github.com/boost-ext/ut.git
    revision=HEAD
    ```
    - `ut` (Git submodule in this repository)
- `meson.build` (file)
```
project('boost.ut meson test', 'cpp')

boostut_proj = subproject('boost.ut')
boostut_dep = boostut_proj.get_variable('boostut_dep', dependencies : boostut_dep)

executable('test.cpp', dependencies : boostut_dep)
```

(I'm not familiar with CI so I'm not sure if such a configuration is possible, sorry)